src/data/aktivitetskrav/aktivitetskravTypes.ts

```diff
@@ -87,3 +87,5 @@ export interface SendForhandsvarselDTO {
 export interface NewAktivitetskravDTO {
   previousAktivitetskravUuid: string;
 }
+
+export type Brevmal = "MED_ARBEIDSGIVER" | "UTEN_ARBEIDSGIVER";
```

Comment:
Er det noe forskjellig mellom dette og `enum`? 🤔 Når vil man bruke det ene vs det andre? 😊

Reply:
Godt spørsmål. Tror det er mye smak og behag og man kan bruke de om hverandre, føler selv at string literal er litt enklere. Og så er det vel noe forskjell i hvordan JS-en blir. Ref: https://brockherion.dev/blog/posts/enums-vs-typed-strings-in-typescript/

Reply:
Så siden enums er en typescript-greie (?), så "koster" det å kompilere det til javascript, mens string literals ikke gjør det? Det er jo i så fall en forbedring 🤔
Føler litt på at vi skriver rene strings i feks `<option value="MED_ARBEIDSGIVER">`, da ville jeg heller sett `<option value={Brevmal.MED_ARBEIDSGIVER}>`. Det gjelder kanskje mest i github, vil tro en IDE highlighter der litt bedre 👍🏼

Reply:
Tenkte også spesifikt på det Eirik nevner over. Blir ikke like lett å skjønne at vi faktisk har en type på det når det er rene strings

Reply:
Ja, man får jo fin autocomplete av IDE når man bruker de der hvor man sender de inn i funksjoner hvor man har definert en parameter av den typen f.eks., men enig i at det er litt verre i <option> som ikke er typet :thinking: